app/views/articles/_medium_featured.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="col-md-6">
+      <div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative">
+        <div class="col p-4 d-flex flex-column position-static">
+          <h3 class="mb-0"><%= article.title%></h3>
+            <div class="mb-1 text-muted"><%= day_month_year(article.created_at) %></div>
+          <p class="card-text mb-auto"><%= truncate(article.body, length: 150) %></p>
+          <%= link_to 'Continue reading...', article_path(article), class: 'text-dark' %>
+        </div>
+        <div class="col-auto d-none d-lg-block">
+          <svg class="bd-placeholder-img" width="200" height="250" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: Thumbnail" preserveAspectRatio="xMidYMid slice" focusable="false"><title>Placeholder</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">Thumbnail</text></svg>
+
+        </div>
+      </div>
+    </div>
```

Comment:
```suggestion
<div class="col-md-6">
  <div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative">
    <div class="col p-4 d-flex flex-column position-static">
      <h3 class="mb-0"><%= article.title%></h3>
      <p class="mb-1 text-muted"><%= day_month_year(article.created_at) %></p>  
      <p class="card-text mb-auto"><%= truncate(article.body, length: 150) %></p>
      <%= link_to 'Continue reading...', article_path(article), class: 'text-dark' %>
    </div>
    <div class="col-auto d-none d-lg-block">
      <svg class="bd-placeholder-img" width="200" height="250" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: Thumbnail" preserveAspectRatio="xMidYMid slice" focusable="false"><title>Placeholder</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">Thumbnail</text></svg>
    </div>
  </div>
</div>
```

Reply:
pq vc está utilizando a class "card-text"?

Pela doc ele parece ser uma class "semântica" ela não possui estilização

Incluir a sugestão da data em uma tag p ao invés de uma div, mas talvez ela poderia está dentro da tah  H3  com estilização menor a direita

Reply:
Estou utilizando  o mb-auto para possibilitar que texto dentro da caixa ocupe o espaço vertical disponível, evitando que haja espaço em branco extra na parte inferior da caixa quando o conteúdo do texto for menor. Mas acredito que posso retirar o card-text e deixar o mb-auto que possui uma finalidade.